docs/source/10min_cudf_cupy.ipynb

```diff
@@ -0,0 +1,410 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "CPU times: user 0 ns, sys: 4 ms, total: 4 ms\n",
+      "Wall time: 405 µs\n",
+      "CPU times: user 0 ns, sys: 0 ns, total: 0 ns\n",
+      "Wall time: 570 µs\n",
+      "CPU times: user 2.84 s, sys: 60 ms, total: 2.9 s\n",
+      "Wall time: 2.9 s\n"
+     ]
+    },
+    {
+     "data": {
+      "text/plain": [
+       "cupy.core.core.ndarray"
+      ]
+     },
+     "execution_count": 3,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "%time cola_cupy = cupy.asarray(df['a'].data.mem)\n",
+    "%time cola_cupy = cupy.fromDlpack(df['a'].to_dlpack())\n",
+    "%time cola_cupy = cupy.asarray(df['a'])\n",
+    "type(cola_cupy)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "From here, we can proceed with normal CuPy workflows, such as reshaping the array or getting the diagonal."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "array([[   0,    1,    2, ...,  197,  198,  199],\n",
+       "       [ 200,  201,  202, ...,  397,  398,  399],\n",
+       "       [ 400,  401,  402, ...,  597,  598,  599],\n",
+       "       ...,\n",
+       "       [9400, 9401, 9402, ..., 9597, 9598, 9599],\n",
+       "       [9600, 9601, 9602, ..., 9797, 9798, 9799],\n",
+       "       [9800, 9801, 9802, ..., 9997, 9998, 9999]])"
+      ]
+     },
+     "execution_count": 4,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "reshaped_arr = cola_cupy.reshape(50,200)\n",
+    "reshaped_arr"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "array([   0,  201,  402,  603,  804, 1005, 1206, 1407, 1608, 1809, 2010,\n",
+       "       2211, 2412, 2613, 2814, 3015, 3216, 3417, 3618, 3819, 4020, 4221,\n",
+       "       4422, 4623, 4824, 5025, 5226, 5427, 5628, 5829, 6030, 6231, 6432,\n",
+       "       6633, 6834, 7035, 7236, 7437, 7638, 7839, 8040, 8241, 8442, 8643,\n",
+       "       8844, 9045, 9246, 9447, 9648, 9849])"
+      ]
+     },
+     "execution_count": 5,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "reshaped_arr.diagonal()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Converting a CuPy Array to a cuDF DataFrame or Series\n",
+    "\n",
+    "We can also convert a CuPy `ndarray` to a cuDF `DataFrame` or `Series`, using the same `dlpack` interface."
```

Comment:
Need to fix this.

Reply:
Now that #1804 is closed, is there more to do here besides re-running so that the output is correct?

Reply:
@randerzander see this updated version